OracleGoldenGate/21c/bin/deployment-main.sh

```diff
@@ -30,6 +27,14 @@ function abort() {
 :     "${OGG_HOME:?}"
 [[ -d "${OGG_HOME}"            ]] || abort "Deployment runtime, '${OGG_HOME}'. not found."
 
+:     "${ABORT_ON_USER_SCRIPT_ERRORS:=true}"
+:     "${SETUP_USER_SCRIPTS:=${OGG_HOME}/scripts/setup}"
+[[ -d "${SETUP_USER_SCRIPTS}" ]] || mkdir -p "${SETUP_USER_SCRIPTS}"
+:     "${STARTUP_USER_SCRIPTS:=${OGG_HOME}/scripts/startup}"
+[[ -d "${STARTUP_USER_SCRIPTS}" ]] || mkdir -p "${STARTUP_USER_SCRIPTS}"
+
+SQLPLUS="$(find ${OGG_HOME} -name sqlplus)"
```

Comment:
Since OGG is not the DBMS, any SQL processing should be handled entirely in the user scripts.

Reply:
I will drop off the sqlplus hook.  I assume this also means dropping the script execution of *.sql files?